tests/unit/samples/print_file_author_test_dir/code3.c

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * codestyle.c
+ *
+ * Copyright (C) 2018 Bob Hilson
+ *
+ * This file is used to test the print_files_authors function
+ * at src/maintainers.sh.
+ *
+ */
+
+#include <stdio.h>
+
+void ordinary_function2(void)
+{
+	puts("Just a function...");
+}
+
+int main(int argc, char *args[])
+{
+	int i;
+	printf("%d\n", argc);
+	for (i = 0; i < argc; ++i)
+		printf("%s ", args[i]);
+	return 0;
+}
+
+MODULE_AUTHOR ( "Martha Elsa <martha@opensource.com>" \
+				"Verena Bert <vb@opensource.com>" )
+ORDINARY_MACRO ("Does something...");
+MODULE_AUTHOR ( "Gabi Katinka <gabik@opensource.com>"
+				"Leonie Hildebert <leoni.hildebert@opensource.com>" )
```

Comment:
Btw, I think you should check this file from the kernel:

drivers/media/usb/gspca/sq930x.c

Reply:
We've added an use of '\n' in the new version so now I think it handles the '\n' case correctly according to the sq930x.c file.